053-SentinelAutomatedResponse/Student/Challenge-03.md

```diff
@@ -0,0 +1,69 @@
+
+
+# Challenge 03 - Automated Response </br>
+
+[< Previous Challenge](./Challenge-02.md) - **[Home](../README.md)** 
+
+## Pre-requisites 
+
+Verify your environment </br>
+- Creates an alert based on your user login </br>
+- Creates an Incident per Alert</br>
+- Watchlist is created
+  </br>
+
+
+## Introduction 
+
+Now we have our alert rule running, the SOC team is finding that its just way too 'noisy' because every time an admin logs on, it's generating an Alerts and an Incidents. It's your job to ensure that the Alert must trigger whenever an administrator logs in and create an Incident, AND, the Incident should be automatically closed if the IP address exists in the Watchlist. 
+
+
+## Description
+We want to automatically update your Security Teams channel when the Incident is not automatically closed.
+
+**Close the Incident automatically**
```

Comment:
This looks like it should be a sub-header since it's a set of challenge instructions.

Use 3 hashes here, which is one level below the 2 hash header for "Description" above it. 
"###Close the Incident Automatically"